src/plugins/intel_gpu/include/intel_gpu/op/fully_connected_compressed.hpp

```diff
@@ -29,7 +29,20 @@ class FullyConnectedCompressed : public FullyConnected {
                              const ov::Output<Node> &decompression_scale,
                              const ov::element::Type output_type = ov::element::undefined);
 
+    FullyConnectedCompressed(const OutputVector& inputs,
+                             bool has_zp = true,
+                             bool has_activation_scale = false,
+                             const ov::element::Type output_type = ov::element::undefined);
```

Comment:
I suggest aligning c-tor with others:
```cpp
    FullyConnectedCompressed(const ov::Output<Node> &A,
                             const ov::Output<Node> &B,
                             const ov::Output<Node> &bias,
                             const ov::Output<Node> &w_decompression_scale,
                             const ov::Output<Node> &w_decompression_zero_point,
                             const ov::Output<Node> &a_decompression_scale,
                             const ov::element::Type output_type = ov::element::undefined);
```
When you use it in transform, you can just append `Placeholder` op for wzp if it's missing to avoid ambiguity

Reply:
Hmm could you suggest how I can code this part then? `FullyConnectedCompressed` will be newly created but there are cases where we have zp or not. 
https://github.com/openvinotoolkit/openvino/pull/25372/files#diff-f849c9ad5eaef07d17827311f30c96125dc34fc631d29e82f18394d91e6c09ddR76

Then it would be something like below. Does it looks good? (same for `clone_with_new_inputs` function)
```
        auto new_fc = std::make_shared<op::FullyConnectedCompressed>(dyn_quan->output(0),
                                                                     m_fc->get_input_node_shared_ptr(1),
                                                                     m_fc->get_input_node_shared_ptr(2),
                                                                     m_fc->get_input_node_shared_ptr(3),
                                                                     m_fc->get_has_zp() ? m_fc->get_input_node_shared_ptr(4) : PlaceHolder,
                                                                     true,
                                                                     m_fc->get_output_type());
```

Reply:
Yes, something like that, but `get_has_zp()` looks redundant, you can just check inputs count

Reply:
Hmm actually my point was that now we have two optional fields(`w_zp`, `a_scale`). From inputs count, we don't know whether we have `w_zp` or `a_scale`.

Reply:
I think there's no ambiguity here. FCCompressed may have 4, 5 or 6 inputs:
`{activation, weights, bias, w_s, [w_zp], [a_s]}`
But there's no way to create  `{activation, weights, bias, w_s, a_s}` version. In that case you have to put placeholder for `w_zp` parameter and use c-tor with 6 args.

Reply:
oh, not updated in this patch. I'll update it tomorrow.

Reply:
updated, thanks!